crates/rattler_shell/src/activation.rs

```diff
@@ -352,6 +358,64 @@ impl<T: Shell + Clone> Activator<T> {
         })
     }
 
+    /// Starts a UNIX shell.
+    /// # Arguments
+    /// - `shell`: The type of shell to start. Must implement the `Shell` and `Copy` traits.
+    /// - `args`: A vector of arguments to pass to the shell.
+    /// - `env`: A `HashMap` containing environment variables to set in the shell.
+    /// - `prompt`: Prompt to the shell
+    #[cfg(target_family = "unix")]
+    #[allow(dead_code)]
+    async fn start_unix_shell<T_: Shell + Copy + 'static>(
+        shell: T_,
+        args: Vec<&str>,
+        env: &HashMap<String, String>,
+        prompt: String,
+    ) -> miette::Result<Option<i32>> {
```

Comment:
Please use anyhow instead of miette in rattler

Reply:
@baszalmstra i updated it to use `anyhow`